src/Sylius/Bundle/UserBundle/Resources/serializer/Model.User.yml

```diff
@@ -0,0 +1,24 @@
+Sylius\Component\User\Model\User:
+    exclusion_policy: ALL
+    xml_root_name: sylius_user
+    properties:
+        id:
+            expose: true
+            type: integer
+            groups: [Default, Detailed]
+        username:
+            expose: true
+            type: string
+            groups: [Default, Detailed, Secured]
+        usernameCanonical:
+            expose: true
+            type: string
+            groups: [Detailed]
+        enabled:
+            expose: true
+            type: boolean
+            groups: [Default, Detailed]
+        roles:
+            expose: true
+            type: array
+            groups: [Detailed]
```

Comment:
Since `User` now holds email related fields (namely `email` and `emailCanonical`), shouldn't those also be included in serialization?